homeassistant/components/rfxtrx/__init__.py

```diff
@@ -52,30 +57,28 @@
         ("Temperature", TEMP_CELSIUS),
         ("Temperature2", TEMP_CELSIUS),
         ("Humidity", PERCENTAGE),
-        ("Barometer", ""),
+        ("Barometer", PRESSURE_HPA),
         ("Wind direction", ""),
-        ("Rain rate", ""),
+        ("Rain rate", "mm/h"),
```

Comment:
```py
f"{LENGTH_MILLIMETERS}/{TIME_HOURS}"
```

Reply:
Should it really be fstrings. Soon to be some other opinions here: https://github.com/home-assistant/core/pull/40436#discussion_r493255626

Reply:
Right now we do it like this. Changing it should happen in a separate PR.

Reply:
Okidok